cmd/viam-agent/main.go

```diff
@@ -35,14 +35,17 @@ var (
 
 //nolint:lll
 type agentOpts struct {
-	Config         string `default:"/etc/viam.json"                        description:"Path to machine credentials file"   long:"config"   short:"c"`
-	DefaultsConfig string `default:"/etc/viam-defaults.json"               description:"Path to manufacturer defaults file" long:"defaults"`
-	Debug          bool   `description:"Enable debug logging (agent only)" env:"VIAM_AGENT_DEBUG"                           long:"debug"    short:"d"`
-	UpdateFirst    bool   `description:"Update versions before starting"   env:"VIAM_AGENT_WAIT_FOR_UPDATE"                 long:"wait"     short:"w"`
-	Help           bool   `description:"Show this help message"            long:"help"                                      short:"h"`
-	Version        bool   `description:"Show version"                      long:"version"                                   short:"v"`
-	Install        bool   `description:"Install systemd service"           long:"install"`
-	DevMode        bool   `description:"Allow non-root and non-service"    env:"VIAM_AGENT_DEVMODE"                         long:"dev-mode"`
+	Config                    string `default:"/etc/viam.json"                                                   description:"Path to machine credentials file"   long:"config"   short:"c"`
+	DefaultsConfig            string `default:"/etc/viam-defaults.json"                                          description:"Path to manufacturer defaults file" long:"defaults"`
+	Install                   bool   `description:"Install systemd service"                                      long:"install"`
+	Debug                     bool   `description:"Enable debug logging (agent only)"                            env:"VIAM_AGENT_DEBUG"                           long:"debug"    short:"d"`
+	ViamDir                   string `description:"Use a custom path for agent directories"                      long:"viam-dir"`
+	EnableSyscfgSubsystem     bool   `description:"Enable system configuration management subsystem"             long:"enable-syscfg-subsystem"`
+	EnableNetworkingSubsystem bool   `description:"Enable networking management subsystem"                       long:"enable-networking-subsystem"`
```

Comment:
enable-syscfg-subsystem -> enable-syscfg? since the subsystem part is more or less implied and this will be slightly easier to type out